packages/core/src/button/Button.tsx

```diff
@@ -26,6 +27,19 @@ export interface ButtonProps extends ComponentPropsWithoutRef<"button"> {
    * 'primary' is the default value.
    */
   variant?: ButtonVariant;
+  /**
+   * To show a loading spinner.
+   */
+  isLoading?: boolean;
```

Comment:
This does not follow our standard prop naming, "is" shouldn't be the prefix

Reply:
removed